x-pack/plugins/security/server/authorization/index.ts

```diff
@@ -121,8 +121,6 @@ export function setupAuthorization({
     },
 
     registerPrivilegesWithCluster: async () => {
-      validateFeaturePrivileges(actions, featuresService.getFeatures());
```

Comment:
**note** Once the client-side is re-written, this validation will no longer be necessary, as it was only checking that privileges were subsets of each other. It's safe to remove now since we are merging into a feature branch.

Reply:
**update**: reintroducing this to allow the security plugin to validate that there are no conflicting feature privilege ids.